voidcrew/code/game/objects/items/devices/radio/radio.dm

```diff
@@ -0,0 +1,3 @@
+/obj/item/radio/talk_into_impl(atom/movable/M, message, channel, list/spans, datum/language/language, list/message_mods)
+	. = ..()
+	playsound(src, "sound/effects/walkietalkie.ogg", 20, FALSE)
```

Comment:
Unless im just missing it, you're not deleting the playsound tg edit we currently have for this. We also check for Solgov frequency there (ok seriously wtf is the point of solgov channel who the fuck speaks there?)

Reply:
i probably forgot to remove it

Reply:
we do check for the frequency there but i just left it because i dont care to have it on tg, we should just remove teh solgov channel